Sources/Datadog/RUMMonitor.swift

```diff
@@ -616,7 +616,8 @@ public class RUMMonitor: DDRUMMonitor, RUMCommandSubscriber {
                     RUMContextAttributes.applicationID: context.rumApplicationID,
                     RUMContextAttributes.sessionID: context.sessionID.rawValue.uuidString.lowercased(),
                     RUMContextAttributes.viewID: context.activeViewID?.rawValue.uuidString.lowercased(),
-                    RUMContextAttributes.userActionID: context.activeUserActionID?.rawValue.uuidString.lowercased()
+                    RUMContextAttributes.userActionID: context.activeUserActionID?.rawValue.uuidString.lowercased(),
+                    RUMContextAttributes.serverTimeOffset: self.applicationScope.sessionScope?.viewScopes.last?.serverTimeOffset
```

Comment:
While usage of `.applicationID`, `.sessionID`, `.viewID` and `.userActionID` is tested [here](https://github.com/DataDog/dd-sdk-ios/blob/4f36c8a23f4416237d8b71846caab747c75b1e68/Tests/DatadogTests/Datadog/LoggerTests.swift#L542-L579), there is no coverage for `.serverTimeOffset`. The purpose of `RUMContextAttributes.serverTimeOffset` remains unclear in `Datadog` module without any test defined in `DatadogTests`.

I think we need a separate test case that will check if these attributes are set correctly in `DatadogContext`. It could explain that these 5 RUM attributes are used for integration with other Features in V2 (Logging + SR).

@maxep WDYT of this? (this is first time we're adding an attribute that has no use in V1 Features, hence we need to test it separately)